hummingbot/connector/derivative/binance_perpetual/binance_perpetual_api_order_book_data_source.py

```diff
@@ -51,12 +52,14 @@ async def get_last_traded_prices(self,
 
     async def get_funding_info(self, trading_pair: str) -> FundingInfo:
         symbol_info: Dict[str, Any] = await self._request_complete_funding_info(trading_pair)
+        funding_interval_hours = await self._get_funding_interval_hours(trading_pair)
```

Comment:
probably is better to use a gather so we spawn the two requests at the same time